在 locals.js 中更新汉化翻译

KS-OTO opened the pull request:
- 有关代码空间、预构建的文案汉化，涉及个人设置、仓库设置；
- 有关个人设置的邮箱设置、通知设置的补充汉化；
- 有关仓库主页、议题的细节优化；
- 有关GitHub个人主页-feed流的细节优化；

涉及：https://github.com/KS-OTO/github-chinese/issues/1 ， https://github.com/KS-OTO/github-chinese/issues/2 ， https://github.com/KS-OTO/github-chinese/issues/3 ， https://github.com/KS-OTO/github-chinese/issues/5 ， https://github.com/KS-OTO/github-chinese/issues/6 ， https://github.com/KS-OTO/github-chinese/issues/7 ， https://github.com/KS-OTO/github-chinese/issues/9 ， https://github.com/KS-OTO/github-chinese/issues/11

Commit:
Update zh-CN translation in locals.js
- 有关代码空间、预构建的文案汉化，涉及个人设置、仓库设置；
- 有关个人设置的邮箱设置、通知设置的补充汉化；

涉及：#1 ， #2 ， #3 ， #5 ，  #6

wuyuncheng-26 commented:
粗略看了看，应该是没什么问题的
现在在火车上，到家再细致看看 :)

wuyuncheng-26 commented:
下次请去设置一下 GPG 签名，否则无法正常合并
教程见 https://docs.github.com/zh/authentication/managing-commit-signature-verification

maboloshi commented on locals.js:
```diff
@@ -3380,6 +3385,9 @@ I18N.zh["settings/emails"] = { // 设置 - 电子邮箱
                 "Are you sure you want to remove this email from your account? Once removed, commits attributed to this email address will no longer be associated with your account. One of your other emails will become your primary address.": "您确定要从您的账户中删除此电子邮箱吗？删除后，归因于该电子邮箱地址的提交将不再与您的账户相关联。您的其他电子邮箱之一将成为您的主要地址。",
+                "At least one non-backup email is required.": "至少需要一个非备份电子邮箱",
+            "Backup": "备用电子邮箱",
+            "This email will be used as an additional destination for security-relevant account notifications and can also be used for password resets.": "该电子邮件将用作与账户安全相关通知的额外接收地址，也可用于密码重置。",
```
```suggestion
            "This email will be used as an additional destination for security-relevant account notifications and can also be used for password resets.": "该电子邮箱将用作与账户安全相关通知的额外接收地址，也可用于密码重置。",
```

KS-OTO replied:
是的，我没有注意到邮件的错误使用，确实应该是邮箱

maboloshi commented on locals.js:
```diff
@@ -3431,10 +3439,12 @@ I18N.zh["settings/emails"] = { // 设置 - 电子邮箱
             "Save subscription preferences": "保存订阅首选项",
             "Back to email settings": "返回电子邮件设置",
-            "To stop receiving emails for the topics below, uncheck any topics you don't want to receive, then select Save subscription preferences": "要停止接收以下主题的电子邮件，请取消选中任何不想接收的主题，然后选择“保存订阅首选项”",
+            "To stop receiving emails for the topics below, uncheck any topics you don't want to receive, then click Save subscription preferences": "若要停止接收以下主题的电子邮件，请取消选中您不想接收的主题，然后操作保存",
```
```suggestion
            "To stop receiving emails for the topics below, uncheck any topics you don't want to receive, then click Save subscription preferences": "若要停止接收以下主题的电子邮件，请取消选中您不想接收的主题，然后点击 “保存订阅首选项”",
```

KS-OTO replied:
我是想着顺从用户操作习惯，并简化该内容。

Commit:
在 locals.js 中更新汉化翻译
- 适用于仓库-议题的细节文案优化

涉及：#7

Commit:
在 locals.js 中更新汉化翻译
- 适用于仓库-议题的细节文案优化

涉及：#7

wuyuncheng-26 commented:
@maboloshi 您那边可以强制合并吗

Commit:
在 locals.js 中更新汉化翻译
- 适用于仓库-提交与合并分支的细节文案优化

涉及：#8

maboloshi commented:
还有一些翻译，还没时间校对。 再放放吧

KS-OTO committed:
在 locals.js 中更新汉化翻译
- 补充汉化：我的主页-我的关注/stars列表

涉及：#9

wuyuncheng-26 commented on locals.js:
```diff
@@ -649,10 +649,14 @@ I18N.zh["pubilc"] = { // 公共区域翻译
                 "Lists": "清单",
                     "Search lists": "搜索清单",
+                    "🔮 Future ideas": "超前想法",
+                    "🚀 My stack": "技术栈",
```
个人感觉这个并不需要翻译

KS-OTO replied:
我是出于“能译尽译”的原则。既然咱们这个汉化插件是想帮助英语小白能更好的使用和掌握GitHub这个软件，那么一方面是尽可能的将页面说明类、引导类、功能类文字进行汉化，同时也可以将一些系统缺省/预设值的内容一并进行汉化，以引导小白用户能够轻松理解这块的设计意图，也能在减少认知的负担的同时，为其使用该功能提供一些助益。

wuyuncheng-26 replied:
hmmm 那也许应该加上表情（

KS-OTO replied:
好的，我会在后续提交时补充

KS-OTO committed:
在 locals.js 中更新汉化翻译
- 补充汉化：仓库-主页/以及其他细节汉化

涉及：#11

Commit:
加空格

Signed-off-by: 是小胖呀26 <122529705+wuyuncheng-26@users.noreply.github.com>

Commit:
小修改

Signed-off-by: 是小胖呀26 <122529705+wuyuncheng-26@users.noreply.github.com>

KS-OTO committed:
补充翻译：个人设置-有关电子邮箱配置的汉化 #3
补充汉化：个人设置-有关通知配置的汉化 #6

KS-OTO committed:
Merge branch 'gh-pages' of github.com:KS-OTO/github-chinese into gh-pages

KS-OTO committed:
补充汉化：仓库-议题-细节文案优化 #7
在锁定 issues的场景中将“Too heated” 的直译进行优化
https://github.com/KS-OTO/github-chinese/issues/7#issuecomment-2249839630

KS-OTO committed:
基于合并审查的修改

KS-OTO reviewed (commented):
完成

KS-OTO committed:
Merge branch 'gh-pages' into gh-pages

maboloshi commented on locals.js:
```diff
@@ -172,7 +172,7 @@ I18N.zh["title"] = { // 标题翻译
         "Webhooks": "Web 钩子",
         "Environments": "环境",
-        "Code security & analysis": "代码安全性与分析",
+        "Security &amp; analysis": "代码安全性与分析",
```
```suggestion
        "Security & analysis": "安全性与分析",
```

maboloshi commented:
@KS-OTO 你这种方式挺好的。相关翻译最近我抽时间在斟酌一下👍

KS-OTO committed:
Merge branch 'maboloshi:gh-pages' into gh-pages

maboloshi commented on locals.js:
```diff
@@ -877,10 +881,10 @@ I18N.zh["pubilc"] = { // 公共区域翻译
             "Learn about vigilant mode": "了解警戒模式",
+            "The email in this signature doesn’t match the committer email.": "此签名中的电子邮箱与提交者的电子邮箱不匹配",
+            "No user is associated with the committer email.": "此签名中的电子邮箱在账号设置中不存在",
 
```
```suggestion
            "The email in this signature doesn’t match the committer email.": "此签名中的电子邮箱与提交者的电子邮箱不相符。",
            "No user is associated with the committer email.": "没有用户与提交者的的电子邮箱关联。",

maboloshi commented on locals.js:
```diff
@@ -1701,9 +1705,14 @@ I18N.zh["page-profile"] = { // 个人首页
+            "You unlocked new Achievements! Show them off by including achievements and private contributions in your Profile in": "您通过私人贡献解锁了新成就！通过在您的个人资料中包含私人贡献来展示它们。",
             "Your pins have been updated. Drag and drop to reorder them.": "您的置顶已更新。拖放来重新排列它们。",
             "Your popular repositories will now be shown instead of your pins.": "现在将显示您的热门仓库，而不是您的置顶。",
+            "Only you can see your full profile.": "以下内容仅你个人可见",
+            "You have marked your profile as private, which limits what activity other people can see. ": "您已将您的个人资料标记为私密，为此系统已限制其他人的查看。",
+            "View what others see": "预览他人视角下的主页",
+            "View full profile": "查看完整主页",
+            "Update profile settings ": "更新个人资料设置",
```
```suggestion
            "Only you can see your full profile.": "只有您可以查看完整的个人资料。",
            "You have marked your profile as private, which limits what activity other people can see. ": "您已将个人资料设为私密，这会限制其他人查看您的活动。",
            "View what others see": "以他人视角查看",
            "View full profile": "查看完整个人资料",
            "Update profile settings": "更新个人资料设置",
```

maboloshi commented on locals.js:
```diff
@@ -2853,7 +2862,7 @@ I18N.zh["settings/notifications"] = { // 设置 - 通知
         // Notification center 通知 https://github.com/settings/notifications
             "Default notifications email": "默认邮件通知设置",
-                "Choose where you'd like emails to be sent. You can add more email addresses. Use custom routes to specify different email addresses to be used for individual organizations.": "选择您希望将邮件发送到的邮箱。您可以添加更多电子邮件地址。使用自定义路由来指定用于各个组织的不同电子邮件地址。",
+                "Choose where you'd like emails to be sent. You can add more email addresses. Use custom routes to specify different email addresses to be used for individual organizations.": "请选择您默认接收通知邮件的邮箱。您可以添加多个邮箱地址，并借助自定义路由来配置不同组织的通知邮件所对应的接收邮箱。",
```
```suggestion
                "Choose where you'd like emails to be sent. You can add more email addresses. Use custom routes to specify different email addresses to be used for individual organizations.": "请选择希望接收通知邮件的邮箱。您可以添加多个邮箱地址。可借助自定义路由为不同的组织指定不同的邮箱地址。",
```

maboloshi commented on locals.js:
```diff
@@ -2919,10 +2929,11 @@ I18N.zh["settings/notifications"] = { // 设置 - 通知
+                    "You can send notifications to different": "您可以自定义各组织的通知邮件发送到不同的电子邮箱地址中",
+                    "verified": "（仅限通过验证的邮箱）",
+                    "email addresses depending on the organization that owns the repository.": " ",
+                    "is your current default email for notifications.": "是您目前默认的通知邮箱",
```
```suggestion
                    "You can send notifications to different": "您可借助自定义路由为不同的组织指定不同的邮箱地址",
                    "verified": "（经验证的邮箱）",
                    "email addresses depending on the organization that owns the repository.": "。",
                    "is your current default email for notifications.": "是您目前默认的通知邮箱。",
```

maboloshi replied:
精简了表述

maboloshi commented on locals.js:
```diff
@@ -3380,12 +3391,19 @@ I18N.zh["settings/emails"] = { // 设置 - 电子邮箱
                 "Are you sure you want to remove this email from your account? Once removed, commits attributed to this email address will no longer be associated with your account. One of your other emails will become your primary address.": "您确定要从您的账户中删除此电子邮箱吗？删除后，归因于该电子邮箱地址的提交将不再与您的账户相关联。您的其他电子邮箱之一将成为您的主要地址。",
+                "At least one non-backup email is required.": "至少需要一个非备份电子邮箱",
+            "Backup": "备用电子邮箱",
+            "This email will be used as an additional destination for security-relevant account notifications and can also be used for password resets.": "该电子邮箱将用作与账户安全相关通知的额外接收地址，也可用于密码重置。",
```
```suggestion
                "At least one non-backup email is required.": "至少需要一个非备用电子邮箱",
            "Backup": "备用电子邮箱",
            "This email will be used as an additional destination for security-relevant account notifications and can also be used for password resets.": "该电子邮箱将用作接收账户安全相关通知的额外方式，也可以用于密码重置。",
```

maboloshi commented on locals.js:
```diff
@@ -877,10 +881,10 @@ I18N.zh["pubilc"] = { // 公共区域翻译
             "SSH key Fingerprint:": "SSH 密钥指纹：",
             "Learn about vigilant mode": "了解警戒模式",
+            "The email in this signature doesn’t match the committer email.": "此签名中的电子邮箱与提交者的电子邮箱不匹配",
+            "No user is associated with the committer email.": "此签名中的电子邮箱在账号设置中不存在",
```
```suggestion
            "The email in this signature doesn’t match the committer email.": "此签名中的电子邮箱与提交者的电子邮箱不相符。",
            "No user is associated with the committer email.": "提交者的电子邮箱未与用户相关联。",

maboloshi commented on locals.js:
```diff
@@ -7262,6 +7294,7 @@ I18N.zh["repository/issues"] = { // 仓库 - 议题页面
                     "Create a branch": "创建分支",
                     "for this issue or link a pull request.": "为这个议题或关联一个拉取请求",
                     "When branches are created from issues, their pull requests are automatically linked.": "当从议题中创建分支时，它们的拉取请求会自动关联。",
+                    "You’re receiving notifications because you were assigned.": "因您是该议题的受理人，为此你将接收该议题的最新通知",
```
```suggestion
```

将该词条移步到`I18N.zh["repository/pull_issue_public"]`的`You’re receiving notifications because`附近？
另外，翻译调整为：
```
您收到通知是因为您受理了它。
```

maboloshi commented on locals.js:
```diff
@@ -9511,6 +9544,9 @@ I18N.zh["repository/actions"] = { // 仓库 - 操作页面
                 "Showing runs from all workflows": "显示所有工作流程的运行情况",
+                "Showing all prebuild configuration runs for Codespaces. Learn about": "已展示当前代码空间下的所有预构建的运行情况 ",
+                "prebuilding your codespaces.": "了解更多预构建Wiki",
+                "Codespaces Prebuilds": "预构建代码空间",
```
```suggestion

            "Codespaces Prebuilds": "代码空间预构建",
                "Showing all prebuild configuration runs for Codespaces. Learn about": "显示代码空间所有预构建设置的运行情况。了解",
                "prebuilding your codespaces.": "代码空间预构建。",
```

maboloshi commented on locals.js:
```diff
@@ -13625,17 +13661,26 @@ I18N.zh["repository/settings/codespaces"] = { // 仓库设置 - 代码空间 /<u
-            "Prebuild configurations speed up Codespace creations significantly by pre-executing all the tasks required to build your development environment.": "预构建配置通过预先执行构建开发环境所需的所有任务，大大加快了代码空间的创建。",
+            "Prebuild configurations speed up Codespace creations significantly by pre-executing all the tasks required to build your development environment.": "预构建设置通过预先执行构建开发环境所需的所有任务，大大加快了代码空间的创建。",
             "Learn more about setting up prebuilds": "了解更多关于预构建设置的信息",
+            "See output": "查看结果",
```
```suggestion

            // 状态
                "Currently Running": "正在运行中",
                "Disabled": "禁用",

            "See output": "查看输出结果",
```

maboloshi commented on locals.js:
```diff
@@ -13625,17 +13661,26 @@ I18N.zh["repository/settings/codespaces"] = { // 仓库设置 - 代码空间 /<u
             "Configuration": "设置",
             "Your prebuild will be built from the branch and configuration file selected below. Learn more about": "您的预构建将从下面选择的分支和配置文件中构建。了解更多关于",
-            "prebuild configuration.": "预构建设置。",
+            "prebuild configuration.": "预构建的设置",
```
```suggestion
            "prebuild configuration.": "预构建设置。",
```

maboloshi commented on locals.js:
```diff
@@ -13625,17 +13661,26 @@ I18N.zh["repository/settings/codespaces"] = { // 仓库设置 - 代码空间 /<u
+            "Enable runs":"恢复运行",
+            "Disabled": "禁用",
 
         // 代码空间 /<user-name>/<repo-name>/settings/codespaces
```
```suggestion
        // 代码空间-新建预构建设置 /<user-name>/<repo-name>/settings/codespaces/prebuild_configurations/new
```

maboloshi commented on locals.js:
```diff
@@ -13625,17 +13661,26 @@ I18N.zh["repository/settings/codespaces"] = { // 仓库设置 - 代码空间 /<u
 
         // 代码空间 /<user-name>/<repo-name>/settings/codespaces
             "/ New prebuild configuration": "/ 新建预构建设置",
+            "/ Edit configuration": "编辑预构建设置",
+            "Prebuilds consume storage space that will incur a billable charge. Learn more about": "预构建操作会占用存储空间，这将产生计费费用",
+            "prebuild configurations": "了解代码空间预构建的计费",
```
```suggestion
            // 黄色警告
                "Prebuilds consume storage space that will incur a billable charge. Learn more about": 预构建会占用存储空间，可能会产生费用。了解更多关于",
                "prebuild configurations": "预构建设置",
```

maboloshi commented on locals.js:
```diff
@@ -13666,8 +13711,8 @@ I18N.zh["repository/settings/codespaces"] = { // 仓库设置 - 代码空间 /<u
             "Region availability": "区域可用性",
-            "Reduce prebuild available to only specific regions": "减少仅适用于特定区域的预构建",
-                "By default, your prebuilt image will be available to all regions where codespaces are available and storage costs will apply for each region. You can adjust this to manage your storage usage.": "默认情况下，您的预构建映像将适用于所有代码空间可用的区域，并且每个地区都有存储费用。。您可以调整此项以管理您的存储使用情况。",
+            "Reduce prebuild availability to only specific regions": "限制预构建在特定区域",
+                "By default, your prebuilt image will be available to all regions where codespaces are available and storage costs will apply for each region. You can adjust this to manage your storage usage.": "默认情况下，您的预构建操作将适用于所有代码空间服务的可用区域，但每个地区都将产生存储费用。您可限制代码空间服务的可用区域来控制存储费用",
```
```suggestion
            "Reduce prebuild availability to only specific regions": "仅在特定地区提供预构建",
                "By default, your prebuilt image will be available to all regions where codespaces are available and storage costs will apply for each region. You can adjust this to manage your storage usage.": "默认情况下，您的预构建镜像将在所有提供代码空间的地区可用，并且每个地区都会产生存储费用。您可以对此进行调整，以管理存储空间的使用。",
```

maboloshi commented on locals.js:
```diff
@@ -13625,17 +13661,26 @@ I18N.zh["repository/settings/codespaces"] = { // 仓库设置 - 代码空间 /<u
+            "Prebuild configurations speed up Codespace creations significantly by pre-executing all the tasks required to build your development environment.": "预构建设置通过预先执行构建开发环境所需的所有任务，大大加快了代码空间的创建。",
             "Learn more about setting up prebuilds": "了解更多关于预构建设置的信息",
+            "See output": "查看结果",
+            "Manually trigger": "手动触发",
+            "View runs": "查看运行日志",
+            "Disable runs":"暂停运行",
+            "Enable runs":"恢复运行",
+            "Disabled": "禁用",
```
```suggestion
            "Configuration menu": "设置菜单",
                "Manually trigger": "手动触发",
                "View runs": "查看运行日志",
                "Disable runs":"暂停运行",
                "Enable runs":"恢复运行",
                
            // 删除对话框
                "Are you sure?": "您确定吗？",
                "Are you sure you want to delete this prebuild configuration? Running workflow may fail and templates associated with this configuration will be deleted.": "您确定要删除这个预构建设置吗？运行中的工作流可能会失败，与此设置相关的模板将被删除。",
```

maboloshi commented on locals.js:
```diff
@@ -13625,17 +13661,26 @@ I18N.zh["repository/settings/codespaces"] = { // 仓库设置 - 代码空间 /<u
 
         // 代码空间 /<user-name>/<repo-name>/settings/codespaces
-            "Prebuild configuration": "预构建配置",
+            "Prebuild configuration": "预构建设置",
```
```suggestion
            // 顶部提醒
                "Prebuild configuration created": "预构建设置已创建",
                "Prebuild configuration deleted": "预构建设置已删除",

            "Prebuild configuration": "预构建设置",
```

maboloshi commented on locals.js:
```diff
@@ -3983,14 +4003,15 @@ I18N.zh["settings/codespaces"] = { // 设置 - 代码空间
-            "encrypted": "被加密的环境变量",
-            ". They are available to any codespace you create using repositories with access to that secret.": "。它们可用于您使用访问该机密的仓库创建的任何代码空间。",
-            "There are no Codespaces secrets.": "尚无代码空间机密",
+            "Development environment secrets are environment variables that are encrypted. They are available to any codespace you create using repositories with access to that secret.": "代码空间机密是被加密的环境变量。您可通过在账号设置下配置全局统一的代码空间加密以快捷的将其应用在您名下的任一代码仓库的代码空间中。",
```
```suggestion
            "Development environment secrets are environment variables that are encrypted. They are available to any codespace you create using repositories with access to that secret.": "开发环境机密是加密的环境变量。它们可用于您可访问该机密的仓库创建的任何代码空间。",
```

maboloshi committed:
修正-1

Signed-off-by: 沙漠之子 <7850715+maboloshi@users.noreply.github.com>

maboloshi commented on locals.js:
```diff
@@ -2877,9 +2888,10 @@ I18N.zh["settings/notifications"] = { // 设置 - 通知
 
                 "Participating, @mentions and custom": "参与、@提及和自定义",
-                    "Notifications for the conversations you are participating in, or if someone cites you with an @mention. Also for all activity when subscribed to specific events.": "您正在参与的对话的通知，或者如果有人 @您。也适用于订阅特定事件时的所有活动。",
+                    "Notifications for the conversations you are participating in, or if someone cites you with an @mention. Also for all activity when subscribed to specific events.": "当您参与的对话有新的通知、或有人 @您时、以及您的订阅特定事件有新的变化时您将收到通知。",
```
```suggestion
                    "Notifications for the conversations you are participating in, or if someone cites you with an @mention. Also for all activity when subscribed to specific events.": "参与的对话、或有人 @您、以及您订阅的特定事件所发出的通知。",
```

maboloshi commented on locals.js:
```diff
@@ -649,10 +649,14 @@ I18N.zh["pubilc"] = { // 公共区域翻译
                     "Create a list to organize your starred repositories.": "创建一个清单来组织您的星标仓库。",
                     "⭐️ Name this list": "⭐️ 清单名称",
                     "Write a description": "简单描述",
+                    "to add emoji to the name or description.": "可在名称或描述中添加表情符号", 
```
```suggestion
                    // 小贴士
                        "type : to add emoji to the name or description.": "输入: 在名称或描述中添加表情符号", 
```

maboloshi commented on locals.js:
```diff
@@ -2866,7 +2877,7 @@ I18N.zh["settings/notifications"] = { // 设置 - 通知
             "Subscriptions": "订阅",
                 "Watching": "关注仓库",
-                    "Notifications for all repositories, teams, or conversations you're watching.": "所有您正在关注的仓库、团队或对话的通知。",
+                    "Notifications for all repositories, teams, or conversations you're watching.": "当您关注的仓库、团队或对话有新的变化时您将收到通知。",
```
```suggestion
                    "Notifications for all repositories, teams, or conversations you're watching.": "所有您关注的仓库、团队或对话所发出的通知。",
```

maboloshi committed:
修订-2

Signed-off-by: 沙漠之子 <7850715+maboloshi@users.noreply.github.com>
